src/components/UserInfo.js

```diff
@@ -1,12 +1,19 @@
 export default class UserInfo {
-  constructor({ name, description }) {
+  constructor({ name, description, avatar }) {
     this._name = document.querySelector(name);
     this._description = document.querySelector(description);
+    this._avatar = document.querySelector(avatar);
   }
+
+  changeAvatarImage(avatar) {
+this._avatar.src= avatar;
+  }
```

Comment:
* You need to format the markup (and code) to show **indentation** and to make the code more readable. You can do that with `Prettier` or a combination `SHIFT + ALT + F`